js/formidable.js

```diff
@@ -153,7 +153,12 @@ function frmFrontFormJS() {
 	 * @param {Object} $form
 	 */
 	function enableSaveDraft( $form ) {
-		$form.find( 'a.frm_save_draft' ).css( 'pointer-events', '' );
+		const saveDraftButton = $form[0].querySelector( '.frm_save_draft' );
+		if ( ! saveDraftButton ) {
+			return;
+		}
+		saveDraftButton.disabled            = false;
+		saveDraftButton.style.pointerEvents = '';
```

Comment:
Maybe it's edgy case, but if we have two submit buttons in the form, this will enable only the first one.

![image](https://github.com/user-attachments/assets/c04f9248-10b7-477b-b047-9d74872cae5d)

What about something like:

```suggestion
		const saveDraftButtons = jQuery($form[0]).find( '.frm_save_draft' );
		if ( saveDraftButtons.length === 0 ) {
			return;
		}
		saveDraftButtons.each( (key, item) => {
			jQuery(item).prop( 'disabled', false ).css( 'pointer-events', '' );
		} );
```

You know the code more than me, it's just an idea, what do u think @AbdiTolesa ?

Reply:
@engahmeds3ed Thanks for reviewing this! I have thought of that too when I was writing the fix but having two draft buttons for a single for didn't make any sense to me and I don't think that is something to bother for. So I believe we are fine with this.

Reply:
Yes I think so :) but I consulted Mike here:

https://strategy11.slack.com/archives/C799A2R61/p1721407141219699?thread_ts=1721340597.429319&cid=C799A2R61

and he said:

> Ha. Fair point. Making it support multiple is probably smart even if we don't anticipate it

What do u think?

Reply:
No problem, I have covered that too now.